build.sh

```
@@ -696,263 +677,85 @@ function package_sample_app() {
-    fi
-
-    # Copy ThunderID declarative resource configs
-    if [ -d "$VANILLA_SAMPLE_APP_DIR/thunderid-config" ]; then
-        echo "Copying ThunderID config..."
-        cp -r "$VANILLA_SAMPLE_APP_DIR/thunderid-config" "$DIST_DIR/$VANILLA_SAMPLE_APP_FOLDER/"
-    else
-        echo "Error: thunderid-config directory not found at $VANILLA_SAMPLE_APP_DIR/thunderid-config"
-        echo "  VANILLA_SAMPLE_APP_DIR=$VANILLA_SAMPLE_APP_DIR"
-        echo "  VANILLA_SAMPLE_APP_FOLDER=$VANILLA_SAMPLE_APP_FOLDER"
-        echo "  DIST_DIR=$DIST_DIR"
+    tgz=$(ls "$DIST_DIR"/thunderid-react-vanilla-sample-*.tgz 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)